$OpenBSD: patch-lib_phusion_passenger_ruby_core_enhancements_rb,v 1.1 2015/01/08 21:24:57 jeremy Exp $

SIGKILL is not trappable on ruby 2.2 (it's ignored in earlier ruby
versions).

Backported from upstream commit b50d031e1b76c7f2782f2673b93cc91c8e0e307a

--- lib/phusion_passenger/ruby_core_enhancements.rb.orig	Thu Jan  8 12:50:41 2015
+++ lib/phusion_passenger/ruby_core_enhancements.rb	Thu Jan  8 12:50:56 2015
@@ -153,7 +153,6 @@ module Signal
 			result.delete("QUIT")
 			result.delete("ILL")
 			result.delete("FPE")
-			result.delete("KILL")
 			result.delete("SEGV")
 			result.delete("USR1")
 			result.delete("IOT")
@@ -172,6 +171,7 @@ module Signal
 		# Other stuff that we don't want to trap no matter which
 		# Ruby engine.
 		result.delete("STOP")
+		result.delete("KILL")
 		
 		return result
 	end
